### DISCUSSION TITLE
CLI parameters removed in latest main

### DISCUSSION THREAD

**Discussion by AntonioCuravalea-IIS** [marcelotduarte/cx_Freeze#2246] [Q&A] [answered]
Hello,

I noticed most CLI parameters were removed with the latest merge done on main. My applications' workflows used the CLI to freeze executables, and one of the parameters I used was `--include-path`.

Was this change intentional? If so, I assume the best way forward would be to migrate to a setup script, right?

**Comment by marcelotduarte** [comment, marked as answer]
The initial intention was to add support to pyproject.toml using the same interface, keeping the parameters that refer to the executable (Executable class) and passing additional parameters to the build_exe command. But among these additional parameters, I see that at least two parameters related to the path were missing (--include-path, --default-path). I'll take care of it.
In other words, the intention is opposite to your question, the objective is not to use setup script, but to be able to use pyproject.toml.

See also #1569

**Comment by AntonioCuravalea-IIS** [comment]
I see - for now I've reverted back to stable, 6.15.14.

I was initially building from main because of a circular dependency error with `ttkbootstrap`, but I've managed to fix it with `--packages="ttkbootstrap"`, so it's probably best I stay on stable for now.
